styleguide.css

```diff
@@ -227,17 +226,22 @@ li a {
     position: relative;
 }
 
-input {
+
+input,
+button {
     border: 0.1rem solid #00283C;
     padding: 2rem;
     margin: 2rem 0;
     background: #fff;
     display: block;
     width: 30rem;
+    transition: border-color .3s;
 }
-input:focus {
+input:focus,
+textarea:focus {
     border: 0.1rem solid #bde513;
     outline: none;
+    transition: border-color .3s;
```

Comment:
you probably want this transition on the normal state, not on `:focus` (like you did for `input` and `button`)

Reply:
ok